src/librustc/hir/map/mod.rs

```diff
@@ -1310,6 +1467,136 @@ fn node_id_to_string(map: &Map<'_>, id: NodeId, include_id: bool) -> String {
     }
 }
 
+// FIXME(@ljedrz): replace the NodeId variant
+fn hir_id_to_string(map: &Map<'_>, id: HirId, include_id: bool) -> String {
```

Comment:
I think you should remove this and make its callers call the regular `NodeId` version.

Reply:
I think this should stay but, as with the similar cases, convert `HirId` to `NodeId` and call the old function.

Reply:
Yeah, sure.